### ISSUE TITLE
Sub-zero not showing up under PMS plugins or agents after reinstalling kitana and all working for months

### ISSUE THREAD

**Issue by Memzmt** [pannal/Sub-Zero.bundle#764]
Hi,
Let me start by saying I have been using sub zero and kitana for sometime now with kitana being installed on docker, I recently reinstalled kitana with docker and I'm not sure if that has anything to do with my issue now regarding Sub-Zero not showing on plex agents or plugins anymore even kitana says I don't have any plugins on server but seeing that I have used them for ages already and know that I am placing Sub-Zero.bundle in the correct plugin folder since it did work previously I am going insane trying to figure out why now it does not show up on plex I will include my Sub-Zero logs with this issue I am opening
[com.plexapp.agents.subzero.log](https://github.com/pannal/Sub-Zero.bundle/files/6414425/com.plexapp.agents.subzero.log)
[com.plexapp.system.log](https://github.com/pannal/Sub-Zero.bundle/files/6414461/com.plexapp.system.log)

Please I will be very grateful if someone can help me through my problem, oh and by the way I installed web tools just so I can check if plugins work at all on plex and what do you know plex shows webtools in plugins but not Sub-Zero sadly I rather Sub-Zero then webtools haha so please if anyone knows anything ill be waiting :)

**Comment by Jojesp** [comment]
I have the same problem since a few days back. I just noticed it today. I think it might have happened on the last Plex server update. I hadn't upgraded Katana. 

I know you stopped developing this plugin Pannal but you would save me so much hassle if you could have a look at it. I still use and love this plugin. :)

**Comment by Jojesp** [comment]
Yes, it seems to be the latest plex server version (1.23.0.4459). I downgraded the server and now it works again.

**Comment by Memzmt** [comment]
Oh my... are you telling me that i didn't cause this and it was the server update that did ?!?! Because if so look I'm happy but damn i spent 2 days trying to find a fix and attempt every possible way i knew how, i guess my luck it might have been just a wild coincidence that i was reinstalling kitana then i noticed i can't see the sub-zero plugin can't remember if i updated it but i do recall sub-zero was working before i reinstalled kitana, i will try to downgrade my plex version and see if this is a temporary fix.
"ILL BE BACK"

**Comment by Memzmt** [comment]
Sadly did not work for me rolling back, may i ask which version you rolled back to? Ill give that a shot rolling back to what you used and if that doesn't work i don't know what will :,(

**Comment by pannal** [comment]
Interesting, I've been on 1.23.0.4459 for quite a while (Beta) and SZ works just as intended. How did you install it?

**Comment by Jojesp** [comment]
I was also on that version which autoupdated thru Plex. After that SZ stopped working. I tried the develop branch and same thing, not working. I tried to repair version 1.23.0.4459 without any luck. Then tried to reinstall that version. Nothing worked to I downgraded Plex to the public version 1.22 and it started to work again. The previous versions of 1.23 worked. 

I will give it a try again to install 1.23.0.4459 fresh and see what happens.

**Comment by Jojesp** [comment]
I saw now that there is a new version out (1.23.0.4482). I updated to it and the plugin doesn't work anymore. I can't see it under plugins and Kitana doesn't pick it up anymore. Did a clean install with the same version and still no luck I'm afraid.

**Comment by Jojesp** [comment]
Ok, so I have done some more testing now. I have installed all the 1.23 versions released and none of them show the SZ plugin under Plugins. I can still see it activated in the agents but if I press the settings button, nothing happens and the settings windows doesn't pop up. I have tried the develop-2.6 branch with clean install and it doesn't seem to matter. I have it installed on Windows 2019 Server just so you know. Do you have it installed on windows pannal?

If I go back to the public 1.22.3.4392 it starts working again.

**Comment by pannal** [comment]
Nah I'm using it on Linux. Can I see the subzero log on one of the failing versions?

**Comment by Jojesp** [comment]
Ok, let see if I done this correctly. This is from a fresh install with 1.23.0.4482 and the develop-2.6 plugin.
[com.plexapp.agents.subzero.log](https://github.com/pannal/Sub-Zero.bundle/files/6426758/com.plexapp.agents.subzero.log)

**Comment by pannal** [comment]
I've reported this to the Plex guys, seems like there was a recent change pertaining to Windows especially, that's resulting in the `fsync` import to fail. Stay tuned.

**Comment by pannal** [comment]
There's a fix in the works.

**Comment by Jojesp** [comment]
Thank you very much for the good news. Again, your plugin is the absolute best there is, it's actually the only one I use these days. :)

**Comment by Jojesp** [comment]
Have you heared anything if its fixed?

**Comment by pannal** [comment]
No yet, no. I believe it'll be there, soon.

**Comment by Jojesp** [comment]
Thank you for the info.

For anyone else reading this thread, if you got the same problem in windows just install the latest public version (1.22.3.4523) and the plugin will work fine until a fix is released for version 1.23... Might save you some time in reading thru this thread and trying to understand which version to go for. :)

**Comment by pax0707** [comment]
That worked.

**Comment by Jojesp** [comment]
I just installed the latest beta (1.23.2.4600 ) and it seems to work now in the latest version. At least I can see the plugin but it doesn't show up in Kitana. Need to add some files to see if it downloads anything.

Do you know if it has been fixed pannal?
